bin/src/reexport.rs

```diff
@@ -29,21 +30,85 @@ pub struct Options {
 #[tracing::instrument]
 pub async fn main(opts: Options) -> Result<()> {
     info!("re-exporting image for FOSSA CLI");
+    try_strategies!(&opts; strategy_tarball, strategy_daemon, strategy_registry)
+}
+
+async fn strategy_registry(opts: &Options) -> Result<()> {
+    if opts.target.is_path() {
+        debug!("input appears to be a file path, skipping strategy");
+        return Ok(());
+    }
```

Comment:
If the issue I speak about in my comment re: `try_strategies` is a real problem I think this file also has it too.